### PR TITLE
Add screenshot decorators to selenium tests

### PR DESCRIPTION
**Pull request by OlegZharkov**
Adds `local_screenshotting` to selenium tests, so we can automate screnshot generation for [rule-based upload tutorial](https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/upload-rules/tutorial.html). Uploading to S3 will be done using https://github.com/galaxyproject/ci-testing. GTN will serve images directly directly from our S3 storage.

As we advance automatic screenshotting, we would need to run screenshot generation against external productive server. `local_screenshotting` here means, that these particular screenshots could be produced locally. 

compliments https://github.com/galaxyproject/galaxy/issues/10794

### CODE REVIEW

File: pytest.ini [hunk 1]
```diff
@@ -6,3 +6,4 @@ python_paths = lib
 markers =
   data_manager: marks test as a data_manager test
   tool: marks test as a tool test
+  local_screenshotting: marks test as a screenshot producer
```
Comment (mvdbeek) [inline pytest.ini]: How do you feel about `screenshot` ?
If you want to discrimate local and remote you should add a second `local` and/or `remote` marker.
These markers can be combined arbitrarily.

Comment (OlegZharkov) [reply pytest.ini]: Ok, will do

File: lib/galaxy_test/selenium/test_uploads.py [hunk 2]
```diff
@@ -173,6 +178,8 @@ def test_rules_example_2_list(self):
         # self.screenshot("rules_example_2_5_download_complete")
 
     @selenium_test
+    @pytest.mark.screenshot
```
Comment (jmchilton) [inline lib/galaxy_test/selenium/test_uploads.py]: A lot of the tests generate screenshots. Maybe call it training_screenshot or tutorial_screenshot or something?

Comment (mvdbeek) [reply lib/galaxy_test/selenium/test_uploads.py]: I'd probably add another marker for training/tutorial if it makes sense to subset them.

Comment (OlegZharkov) [reply lib/galaxy_test/selenium/test_uploads.py]: > I'd probably add another marker for training/tutorial if it makes sense to subset them.

I genuinely don't think it's something important, it's rather a minor detail. But I really need it for https://github.com/galaxyproject/ci-testing and if it helps to get it merged...

**Pull request merged**